proxyclient/m1n1/hv/__init__.py

```diff
@@ -1603,8 +1560,6 @@ def load_hook(data, segname, size, fileoff, dest):
         self.tba.devtree = self.adt_base - phys_base + self.tba.virt_base
         self.tba.top_of_kernel_data = guest_base + image_size
 
-        self.sym_offset = macho.vmin - guest_base + self.tba.phys_base - self.tba.virt_base
```

Comment:
@marcan @amarioguy   This commit breaks symbols for XNU kernels, probably because sym_offset is always zero now. I've revered it locally for now.